docs/updates/April_2024.md

```diff
@@ -0,0 +1,108 @@
+---
+sidebar_position: 6
+---
+
+# April 2024
+
+## tl;dr
+
+- Interviewing a number of folks for the front end devs to help maintain
```

Comment:
Probably nit-picking here, but can you please rename `front end` to `Front-end` in another 3 instances as well?
```suggestion
- Interviewing a number of folks for the Front-end devs to help maintain
```